query/releasesearch.go

```diff
@@ -233,28 +233,22 @@ type ReleaseBuilder struct {
 	v710            bool
 }
 
-func NewReleaseBuilder(v710 bool) (*ReleaseBuilder, error) {
+func NewReleaseBuilder() (*ReleaseBuilder, error) {
 	var (
 		searchTemplate *template.Template
 		err            error
 	)
+	searchTemplate, err = template.ParseFS(releaseFS,
+		"templates/releasecalendar/search.tmpl",
+		"templates/releasecalendar/query.tmpl")
```

Comment:
I think you've chosen incorrectly: it should be the v710 templates that are used